js/main.js

```diff
@@ -5208,8 +5227,44 @@ function updateSchedule() {
     updateScheduleForADay(day_1_schedule, tracks, $(".schedule-table-1 tbody"));
     updateScheduleForADay(day_2_schedule, tracks, $(".schedule-table-2 tbody"));
     updateTrackHall(track_halls, '.track-hall');
+
+    for(var i=0; i<talk_count; i++){
```

Comment:
Atleast we can put this into a function for now and call it in `$( document ).ready()` below